nnas

```diff
@@ -2,13 +2,15 @@
 
 NNAS_CONFIG_RPATH="infra/config"
 NNAS_COMMAND_RPATH="infra/command"
+
```

Comment:
this adding empty line doesn't seem to be relevent to this PR explanation.

Reply:
I think unifying blank line convention is different issue. It's better to revert blank line change.

Reply:
Done